kujira/database/mongodb/lib/mongodb_lib.py

```diff
@@ -0,0 +1,51 @@
+'''
+library to acces Mongo database allowing to insert and get tasks.
+'''
+
+import datetime
+from pymongo import MongoClient
+
+class Mongodb(object):
+    '''contains connecting, inserting and geting tasks from database'''
+
+    def __init__(self):
+        self.client = MongoClient()
+        self.database_connection = None
+        self.database_collection = None
+        self.database_collection_history = None
+        self.post = None
+        self.document = None
+
+    def connect(self, data_base_name, collection_name, history_collection_name):
+        '''method setting up the connection to database'''
+        self.database_connection = self.client[data_base_name]
+        self.database_collection = self.database_connection[collection_name]
+        self.database_collection_history = self.database_connection[history_collection_name]
+
+
+    def insert_task(self, name):
+        '''method inserting task(actually just a name) as record into specified
+         collections in database, auto generated date, to allow getting oldest event'''
+        self.post = {"name": name, "date": datetime.datetime.utcnow()}
+        return self.database_collection.insert_one(self.post).inserted_id
+
+
+    def get_task(self):
+        '''gets the newest task from database, deletes it and then inserts into
+        historical collection with beginning state "doing"'''
+        temporary_cursor = (self.database_collection.find().sort("date", 1).limit(1))
```

Comment:
1. get_task() should just return first (last inserted) task.
2. new method to update status of task 'set_status'
3. new method close_task to add task to tasks_history collection, set its status and result, then remove task from tasks collection